aries-site/src/themes/aries.js

```diff
@@ -7,9 +7,109 @@ export const aries = deepMerge(hpe, {
   // To be stripped out once theme changes are made in grommet-theme-hpe.
   // keeping file for use as playground for future theme adjusments that need
   // to be quickly tested
+  pageHeader: {
```

Comment:
we can revert pageHeader this was added for testing purposes 
https://github.com/grommet/grommet-theme-hpe/pull/536